presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergColumnHandle.java

```diff
@@ -160,6 +164,59 @@ public String toString()
         return getId() + ":" + getName() + ":" + type.getDisplayName() + ":" + columnType + ":" + requiredSubfields;
     }
 
+    public static IcebergColumnHandle pathColumnHandle()
+    {
+        return getIcebergColumnHandle(FILE_PATH);
```

Comment:
Should these not just be constants?  And if so, let's just make them public static, since IcebergColumnHandle is immutable.

Reply:
+1